### PR TITLE
Remove opencv

### LINKED ISSUE #78: Remove dependency opencv-python
OpenCV is no longer used and should be removed. See https://github.com/leng-yue/py-scrcpy-client/pull/65

### PR DESCRIPTION
**Pull request by aentwist**
See commits

Closes #78 ,#74

Related to #62

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -16,7 +16,7 @@ py-scrcpy = "scrcpy_ui:main"
 
 [tool.poetry.dependencies]
 # Should match the python versions in .github/workflows/ci.yml and publish.yml
```
Comment (aentwist) [inline pyproject.toml:18]: you fool

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -15,8 +15,8 @@ jobs:
     strategy:
       matrix:
         # Should match the python versions in pyproject.toml and publish.yml
-        python-version: ["3.10"]
-        os: [ubuntu-latest]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
+        os: [windows-latest, ubuntu-latest, macos-latest]
```
Comment (aentwist) [inline .github/workflows/ci.yml:19]: @leng-yue I see you changed this earlier, is there anything I should be aware of? I'll at least try to get things passing like this.

Comment (leng-yue) [reply .github/workflows/ci.yml:19]: Seems there was some issue on 3.12

**Comment by aentwist** [comment]
LFG son

**Comment by aentwist** [comment]
@leng-yue up to you

**Pull request merged**